microsim/opencl/ramp/kernels/ramp_ua.cl

```diff
@@ -116,7 +116,7 @@ float get_obesity_multiplier(ushort obesity, global const Params* params){
 }
 
 bool is_overweight(ushort obesity){

```

Comment:
it might make sense now to rename this function to `is_obese`, totally optional though

Reply:
Good idea - would definitely make things clearer

Reply:
done!

Reply:
nice, this all looks good to me then, happy for this to be merged!